README.md

```diff
@@ -27,7 +21,19 @@ If you prefer to do it straight from CLI here is a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
 
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it the supported way:
+```bash
+go get -u -d github.com/redhat-developer/opencompose
+make install -C $GOPATH/src/github.com/redhat-developer/opencompose/
+```
+This will make sure proper output of `opencompose version` which is dependant on advanced build.
```

Comment:
"... is dependant on advanced build." - hmm, I don't know what that sentence means :-(